src/imf_reader/sdr/read_announcements.py

```diff
@@ -73,14 +73,16 @@ def get_holdings_and_allocations_data(
 
 
 @lru_cache
-def get_latest_allocations_holdings_date() -> tuple[int, int]:
+def get_latest_allocations_holdings_date(log_info=True) -> tuple[int, int]:
```

Comment:
Just noticed, this should be called "fetch..." to be consistent with the other user facing functions